CS_UCDs.ipynb

```diff
@@ -248,9 +253,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python [conda env:pyvo-dev]",
    "language": "python",
-   "name": "python3"
+   "name": "conda-env-pyvo-dev-py"
```

Comment:
The continuous integration check did not like the kernel name.

`[nbcollection (ERROR)]: Notebook 'CS_UCDs.ipynb' errored: No such kernel named conda-env-pyvo-dev-py`